DataRepo/views/models/bst/column/field.py

```
@@ -139,8 +139,17 @@ def __init__(
             )
 
         self.field = field_path_to_field(self.model, self.field_path)
-        if not hasattr(self, "is_fk") or getattr(self, "is_fk", None) is None:
-            self.is_fk = is_key_field(self.field)
```

C. Foreign keys are automatically created using `BSTRelatedColumn`, but some tests were using `BSTColumn`.  Only `BSTRelatedColumn` and its derived classes handle representative fields, so I decided I would raise an exception if `BSTColumn` was used on a foreign key.